auth/services/oauth/oauth.go

```diff
@@ -155,6 +155,29 @@ func (s *service) validateActor(context *validationContext) error {
 	return nil
 }
 
+// check if the aud service identifier matches the oauth endpoint of the requested scope
+func (s *service) validateAud(context *validationContext) error {
```

Comment:
rest of the func names are writtng in full, so let's just say `validateAudience`